src/notation/internal/notationinteraction.cpp

```diff
@@ -876,6 +876,16 @@ void NotationInteraction::startDrag(const std::vector<EngravingItem*>& elems,
     qreal proximity = configuration()->selectionProximity() * 0.5f / zoom;
     m_scoreCallbacks.setSelectionProximity(proximity);
 
+    if (elems.size() == 1 && (m_editData.modifiers & Qt::KeyboardModifier::AltModifier)) {
+        if (elems[0]->isSpannerSegment()) {
+            auto copy = toSpannerSegment(elems[0])->spanner()->clone();
+            score()->undoAddElement(copy);
+        } else {
+            auto copy = elems[0]->clone();
+            score()->undoAddElement(copy);
```

Comment:
there are some element types (e.g. accidentals, rests) where this doesn't really work properly and might cause a memory-leak or potentially odd behaviour, though from testing I didn't notice anything except that you simply don't see the copied element.